src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -65,6 +80,14 @@ public void setMotorSpeedShoot(double speed) {
     rightMotor.set(ControlMode.PercentOutput, speed);
   }
 
+  public void setInstantRumble() {
+    conDriver.setRumble(RumbleType.kBothRumble, prefControllers.rumbleOutput.getValue());
+    conOperator.setRumble(RumbleType.kBothRumble, prefControllers.rumbleOutput.getValue());
+    Commands.waitSeconds(prefControllers.rumbleDelay.getValue());
```

Comment:
I don't think a Command.waitSeconds() works in this context, I would recommend using rio FPGA timer (https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Timer.html#getFPGATimestamp())